src/index.js

```diff
@@ -0,0 +1,157 @@
+    tempValue: null, 
+    landscape: null,
+    cityNameOutput : null,
+    cityNameInput: null,
+    currentWeatherButton : null,
+    resetButton : null,
+    skySelect : null,
+    sky : null,
+};
+
+const setDefaultValues = () => {
+    state.cityNameInput.value = `${state.cityName}`;
+    state.cityNameOutput.textContent = `${state.cityName}`;
+    state.landscape.textContent = `${state.defaultLandscape}`;
+    tempValue.style.color = state.defaultColor;
+    tempValue.textContent = 0;
+    sky.textContent = "";
+};
+
+const registerEventHandlers = () => {
+    state.increaseTempButton.addEventListener("click", increaseTemp, changeColor, changeLandscape);
+    state.decreaseTempButton.addEventListener("click", decreaseTemp, changeColor, changeLandscape);
+    state.currentWeatherButton.addEventListener("click", displayCurrentWeather);
+    state.cityNameInput.addEventListener("input", displayCityName); 
+    state.resetButton.addEventListener("click", setDefaultValues);
+    state.skySelect.addEventListener("change", changeSky);
+};
+
+const loadControls = () => {
+    state.increaseTempButton = document.getElementById("increaseTempControl");
+    state.decreaseTempButton = document.getElementById("decreaseTempControl");
+    state.cityNameInput = document.getElementById("cityNameInput");
+    state.cityNameOutput = document.getElementById("headerCityName");
+    state.currentWeatherButton = document.getElementById("currentTempButton");
+    state.tempValue = document.getElementById("tempValue"); 
+    state.landscape = document.getElementById("landscape");
+    state.resetButton = document.getElementById("cityNameReset");
+    state.skySelect = document.getElementById("skySelect");
+    state.sky = document.getElementById("sky");
+};
+
+const increaseTemp = () => {
+    state.tempDeg += 1;
+    tempValue.textContent = `${state.tempDeg}`
+    changeColor();
+    changeLandscape();
+};
+
+const decreaseTemp = () => {
+    state.tempDeg -= 1;
+    tempValue.textContent = `${state.tempDeg}`;
+    changeColor();
+    changeLandscape();
+};
+
+const changeColor = () => {
+    let currentTemp = tempValue.textContent;
+    if (currentTemp >= 80) {
+        tempValue.style.color = "red";
+    } else if (currentTemp >= 70) {
+        tempValue.style.color = "orange";
+    } else if (currentTemp >= 60) {
+        tempValue.style.color = "yellow";
+    } else if (currentTemp >= 50) {
+        tempValue.style.color = "green";
+    } else {
+        tempValue.style.color = "teal";
+    }
+};
+
+const changeLandscape = () => {
+    let currentTemp = tempValue.textContent;
+    if (currentTemp >= 80) {
+        landscape.textContent = `${state.hotLandscape}`;
+    } else if (currentTemp >= 70) {
+        landscape.textContent = `${state.niceLandscape}`;
+    } else if (currentTemp >= 60) {
+        landscape.textContent = `${state.midLandscape}`;
+    } else {
+        landscape.textContent = `${state.defaultLandscape}`;
+    }
+};
+
```

Comment:
Another way to format these two methods would be to use an object. The keys of the object could be temp values and then you could have values be colors (for `changeColor()`) or landscape emojis (for `changeLandscape()`). 

```javascript
const colorObject = {80: 'red', 70: 'orange', 60: 'yellow'} // etc 
```

When you have an object, you don't need to do if/else if/else if/ else which can introduce bugs and also adds complexity to the method. If you have an object, then you can iterate over the object and check the keys with currentTemp and then get the correct values to set the color and landscape emojis.

Also, on lines 85-91, you shouldn't need to use string templates to get the values from the state object. You should be able to just do something like `landscape.textContent = state.niceLandscape;`